docs/source/common/lightning_module.rst

```diff
@@ -918,21 +915,19 @@ The current epoch
         if self.current_epoch == 0:
             ...
 
--------------
```

Comment:
Are we removing these because we are  updating the styling and spacing globally in the docs?

So far @williamFalcon  strongly insisted in this particular formatting style, so we should keep it unless clearly agreed on.

Reply:
yes.. mostly the top-level headings are separated by these breaks. Adding it to all of these individual properties adds up a lot of space/padding in between. Without them, it looks good to me.
<img width="875" alt="Screenshot 2022-01-10 at 1 46 23 AM" src="https://user-images.githubusercontent.com/30778939/148699206-b4ad5137-e955-4291-bc70-55c3a6115f19.png">